taskcluster/ci/signing/kind.yml

```diff
@@ -16,20 +16,29 @@ primary-dependency: build
 
 group-by: build-type
 
+only-for-build-types:
+    # - macosx/opt
+    - android-x64/release
+    - android-x86/release
+    - android-arm64/release
+    - android-armv7/release
+
 job-template:
-    description: Sign MozillaVPN for Android
+    description: Sign MozillaVPN
     worker: {} # Will be configured by signing:transforms
-    signing-format: autograph_apk
-    run-on-tasks-for: [github-push]
```

Comment:
Actually, I think this needs to be an empty list because otherwise it will default to "all":
https://hg.mozilla.org/ci/taskgraph/file/tip/src/taskgraph/target_tasks.py#l44